packages/xrpl/test/client/getOrderbook.test.ts

```diff
@@ -37,50 +41,56 @@ function isBTC(currency: string): boolean {
   )
 }
 
-function normalRippledResponse(
-  request: BookOffersRequest,
-): Record<string, unknown> {
+function normalRippledResponse(request: Request): Record<string, unknown> {
```

Comment:
Why can't the type of `request` just be a `BookOffersRequest`?

Reply:
@JST5000 Because `addResponse` from the mock server only accepts a pure `Request` as the typescript type. I could modify it to be a template definition, but it doesn't seem worth it versus just doing type coalescing.